src/status_im/common/metrics_confirmation_modal/view.cljs

```diff
@@ -0,0 +1,74 @@
+(ns status-im.common.metrics-confirmation-modal.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.common.metrics-confirmation-modal.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- hide-bottom-sheet
+  []
+  (rf/dispatch [:hide-bottom-sheet]))
+
+(defn- toggle-metrics
+  [enabled?]
+  (rf/dispatch [:centralized-metrics/toggle-centralized-metrics enabled?]))
+
+(def will-receive-points
+  [(i18n/label :t/ip-address)
+   (i18n/label :t/universally-unique-identifiers-of-device)
+   (i18n/label :t/logs-of-actions-withing-the-app)])
+
+(def not-receive-points
+  [(i18n/label :t/your-profile-information)
+   (i18n/label :t/your-addresses)
+   (i18n/label :t/information-you-input-and-send)])
+
+(defn- render-points
+  [{:keys [title points]}]
+  [rn/view
+   [quo/text {:weight :semi-bold}
+    title]
+   (map-indexed
+    (fn [idx label]
+      ^{:key (str idx label)}
+      [quo/markdown-list
+       {:description     label
+        :blur?           true
+        :container-style style/item-text}])
+    points)])
+
+(defn view
+  [{:keys [settings?]}]
+  (let [on-share-usage  (rn/use-callback
+                         (fn []
+                           (toggle-metrics true)
+                           (hide-bottom-sheet)))
+        on-do-not-share (rn/use-callback
+                         (fn []
+                           (toggle-metrics false)
+                           (hide-bottom-sheet)))]
+    [rn/view
+     [quo/drawer-top
+      {:title       (i18n/label :t/help-us-improve-status)
+       :description (i18n/label :t/collecting-usage-data)}]
+     [rn/view {:style style/points-wrapper}
+      [render-points
+       {:title  (i18n/label :t/what-we-will-receive)
+        :points will-receive-points}]
+      [render-points
```

Comment:
Nit: components are named as nouns and don't start with verbs because this discourages the confusion in hiccup to call them as functions, which can cause bugs on rerender (not the case here of course), but that's the idea.